### ISSUE TITLE
I am using fontAwesome icons for edit,save,delete,cancel .But only the addNew icon i.e(plus symbol) is not showing on screencouldn't show icon for AddNew button.If I use the plus icon inplace of save/delete/edit it is showing up but the problem is with addNew .Can someone please give me a solution to fix this?

### ISSUE THREAD

**Issue by lakshmimahankali** [akveo/ng2-smart-table#1073]

**Comment by lakshmimahankali** [comment]
I am using fontAwesome icons for edit,save,delete,cancel .But only the addNew icon i.e(plus symbol) is not showing on screencouldn't show icon for AddNew button.If I use the plus icon inplace of save/delete/edit it is showing up but the problem is with addNew .Can someone please give me a solution to fix this?

I used the following settings
this.settings = {
                add: {
                    confirmCreate: true,
                    addButtonContent: '<i class="fa fa-plus"></i>',
                    createButtonContent: '<i class="fa fa-save"></i>',
                    cancelButtonContent: '<i class="fa fa-remove"></i>'
                },
                delete: {
                    confirmDelete: true,
                    deleteButtonContent: '<i class="fa fa-trash"></i>'
                },
                edit: {
                    confirmSave: true,
                    editButtonContent: '<i class="fa fa-edit"></i>',
                    saveButtonContent: '<i class="fa fa-save"></i>',
                    cancelButtonContent: '<i class="fa fa-remove"></i>'
                }
      }

**Comment by nvmnghia** [comment]
Check if you included the CSS for fontawesome

**Comment by lakshmimahankali** [comment]
yes I included.Edit,save,delete icons are displaying using fontawesome only.But only plus button is not showing.If I put plus icon instaed of delete or edit or save it is showing on screen
